docs/data/date-pickers/date-field/DebouncedDateField.js

```diff
@@ -1,36 +1,34 @@
 import * as React from 'react';
-import addWeeks from 'date-fns/addWeeks';
+import dayjs from 'dayjs';
 import { debounce } from '@mui/material/utils';
 import { LocalizationProvider } from '@mui/x-date-pickers';
-import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
 import { Unstable_DateField as DateField } from '@mui/x-date-pickers/DateField';
 import Stack from '@mui/material/Stack';
 import Typography from '@mui/material/Typography';
 import Button from '@mui/material/Button';
-import format from 'date-fns/format';
-import isEqual from 'date-fns/isEqual';
 
-const today = new Date();
+const today = dayjs();
 
 export default function DebouncedDateField() {
-  const [value, setValue] = React.useState(() => addWeeks(today, 1));
+  const [value, setValue] = React.useState(() => today.add(1, 'week'));
 
   const debounceSetValue = React.useMemo(() => debounce(setValue, 500), []);
 
   return (
     <Stack spacing={2}>
       <Typography>
-        Value outside the field: {value == null ? 'null' : format(value, 'P')}
+        Value outside the field: {value == null ? 'null' : value.format('P')}
```

Comment:
This should probably be `dayjs` valid localized format 🤔 
But [documentation mentions](https://day.js.org/docs/en/display/format) that for it to work `localizationFormat` plugin has to be used...
```suggestion
        Value outside the field: {value == null ? 'null' : value.format('LT')}
```